src/main/resources/templates/sparql/organization/peopleTitle.sparql

```diff
@@ -20,7 +21,7 @@ CONSTRUCT {
         ?position a vivo:Position .
         ?position vivo:relatedBy ?person .
         ?person a foaf:Person .
-        ?person vivo:hrJobTitle ?title .
-        BIND( CONCAT( STR(?title), REPLACE(STR(?person), "(^.*/)", "::") ) AS ?titleWithId ) .
+        ?position rdfs:label ?label .
+        BIND( CONCAT( STR(?label), REPLACE(STR(?person), "(^.*/)", "::") ) AS ?labelWithId ) .
     }
 }
```

Comment:
Traling new line not present.

Reply:
Yep, I will get them soon enough. There are a lot of them without and will do them all at once.

Reply:
I also just noticed that there are comments on the PR itself about the newline.
I reviewed the code but I didn't think to look at the comments that are not in code review part of the PR.